repos/exporter/lib/exporter-stack.ts

```diff
@@ -110,6 +110,39 @@ export class ExporterStack extends cdk.Stack {
             targets: [new LambdaFunction(atcoCodeCheckerFunction)],
         });
 
+        const tableRenameFunction = new NodejsFunction(this, `table-rename-${stage}`, {
+            functionName: `table-rename-${stage}`,
+            entry: './lib/tableRenameHandler.ts',
+            environment: {
+                RDS_HOST: Fn.importValue(`${stage}:RdsClusterInternalEndpoint`),
+                STAGE: stage,
+            },
+            securityGroups: [
+                SecurityGroup.fromSecurityGroupId(
+                    this,
+                    'table-rename-security-group',
+                    Fn.importValue(`${stage}:ReferenceDataUploaderLambdaSG`),
+                ),
+            ],
+            vpc: vpc,
+            vpcSubnets: vpcSubnets,
+            bundling: {
+                sourceMap: true,
+                sourceMapMode: SourceMapMode.DEFAULT,
+            },
+            timeout: Duration.minutes(15),
+            logRetention: 180,
+        });
+
+        this.addAlarmsToLambda(stage, tableRenameFunction, `tableRenameFunction-${stage}`, 420000);
+
+        tableRenameFunction.addToRolePolicy(new PolicyStatement({ actions: ['ssm:GetParameter'], resources: ['*'] }));
```

Comment:
Can the resources be restricted here and explicitly list only the parameters it needs, rather than `*`?